### PR TITLE
remake user with devise

### PR DESCRIPTION
**Pull request by jess-nash**
Sorry 😭

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  devise_for :users
```
Comment (TyJHennig) [inline config/routes.rb:2]: Thats WRONG!!!!!!!!

Comment (jess-nash) [reply config/routes.rb:2]: :(

**Review by TyJHennig — APPROVED** [review]
LGTM

**Comment by TyJHennig** [comment]
@noliver-ykc @k-ashmore

**Review by k-ashmore — APPROVED** [review]
LGTM

File: config/routes.rb (hunk 1 above)
Comment (TyJHennig) [reply config/routes.rb:2]: (RYAN TOLD ME TO .·´¯`(>▂<)´¯`·.  )

**Pull request merged**